### PR TITLE
Fix: Extract Accordion Data into an Array

### LINKED ISSUE #26: extract accordion data into an array
https://github.com/aaron-org/snapit-hacktoberfest/blob/d13b3b6ac5ee5b963956eaf8b0e204f13ac6aa9b/src/components/AccordionList.jsx#L27-L46

Is it possible to extract the data for each accordion item into an array of objects, then loop through the object to display the data?

### PR DESCRIPTION
**Pull request by camitaylor**
# [Fix: Extract Accordion Data into an Array](https://trello.com/c/AqccUMDe/8-create-faq-page)

## A short description of what this PR does.
As a Developer, I want the AccordionList data to be in an array and the data mapped through to display.
Fixes #26 

## How was it tested?
This was linted with Yarn and tested with Jest.

Screen Shot 2021-09-27 at 2 04 04 PM

### Yarn Lint

Screen Shot 2021-09-27 at 2 04 26 PM

### Checklist
- [x] I acknowledge that all my contributions will be made under the project's license
- [x] I have made a material change to the repo (functionality, testing, spelling, grammar)
- [x] I have read the [Contribution Guidelines](https://github.com/aaronshivers/snapit-hacktoberfest/blob/main/CONTRIBUTING.md) and my PR follows them
- [x] I have titled the PR appropriately
- [x] I have updated my branch with the main branch
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added the necessary documentation about the functionality in the appropriate .md file
- [x] I have added inline documentation to the code I modified

### CODE REVIEW

File: src/components/AccordionList.jsx [hunk 1]
```diff
@@ -4,133 +4,125 @@ import AccordionSummary from "@mui/material/AccordionSummary";
...
 import ExpandMoreIcon from "@mui/icons-material/ExpandMore";
-import { FormattedMessage } from "react-intl";
+import { useIntl } from "react-intl";
 
-export default function SimpleAccordion() {
-  return (
+const AccordionList = () => {
+  const intl = useIntl();
+
+  const AccordionData = [
```
Comment (aaronshivers) [inline src/components/AccordionList.jsx:12]: 🔧 Could you put this into its own file in the same directory.

File: src/lang/en-US.json [hunk 2]
```diff
@@ -3,6 +3,12 @@
   "Countdown-fourth-list-item": "SECONDS",
   "Countdown-second-list-item": "HOURS",
   "Countdown-third-list-item": "MINUTES",
+  "FAQ-list-answer1": "Lorem ipsum dolor sit amet, consectetur adipiscing elit  Suspendisse malesuada lacus ex, sit amet blandit leo lobortis eget.",
```
Comment (aaronshivers) [inline src/lang/en-US.json:6]: :wrench: We don't need six identical translations.

**Review by aaronshivers — CHANGES_REQUESTED** [review]

File: src/components/AccordionList.jsx (hunk 1 above)
Comment (camitaylor) [reply src/components/AccordionList.jsx]: Addressed: commit 9ed5a025e23ab2e934a5c8e03db5fa335555f6e3

File: src/lang/en-US.json (hunk 2 above)
Comment (camitaylor) [reply src/lang/en-US.json]: Addressed: commit ee174247904fe884c47cfb36e579dd8e0ed3d65e

File: src/components/AccordionList.jsx (hunk 1 above)
Comment (aaronshivers) [reply src/components/AccordionList.jsx]: :wrench: Can you make this a `js` file?

**Review by aaronshivers — APPROVED** [review]

**Pull request merged**